### ISSUE TITLE
Game Settings menus will time out.

### ISSUE THREAD

**Issue by fisherKat289** [jakelauer/allbadcards#276]
URL: [https://allbad.cards/game/blue-collar-alligator-293](url)

As I edit the decks or am adding a list of decks in, the UI just closes. It seems like it times out.

I didn't see any actual errors.

This is on Chrome on Mac.

**Comment by jakelauer** [comment]
I haven't experienced this or seen other reports. Have you seen this happen since posting?

**Comment by dennmuel** [comment]
I can confirm the behaviour for the time this issue was opened. I've tried it again now, but instead of the UI closing immediately I see the lost connection message:
![grafik](https://user-images.githubusercontent.com/21078769/82992551-9a27d500-9fff-11ea-816d-8a7359fc4d21.png)
On clicking retry, the settings menu I opened had closed. I can only assume both phenomena have the same cause, because the timing seems the same.
